### ISSUE TITLE
1.3.20 EAP2 broken on Windows

### ISSUE THREAD

**Issue by msink** [JetBrains/kotlin-native#2482]
1.3.20 EAP1 works, EAP2 works on Linux and Mac, but broken on Windows:

https://github.com/msink/kotlin-libui/commits/kotlin-1.3.20

https://ci.appveyor.com/project/msink/kotlin-libui/builds/21145364

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: C:\Users\appveyor\.konan\dependencies\msys2-mingw-w64-x86_64-gcc-7.3.0-clang-llvm-lld-6.0.1\bin\libclang.dll: Can't find dependent libraries
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.jetbrains.kotlin.native.interop.gen.jvm.MainKt.prepareTool(main.kt:306)
	at org.jetbrains.kotlin.native.interop.gen.jvm.MainKt.processCLib(main.kt:192)
	at org.jetbrains.kotlin.native.interop.gen.jvm.MainKt.interop(main.kt:36)
	at org.jetbrains.kotlin.cli.utilities.InteropCompilerKt.invokeInterop(InteropCompiler.kt:88)
	at org.jetbrains.kotlin.cli.utilities.MainKt.main(main.kt:18)
> Task :libui:cinteropLibuiWindows FAILED
FAILURE: Build failed with an exception.
```

**Comment by olonho** [comment]
Are you using 64-bit machine and JDK?

**Comment by msink** [comment]
Yes, both on my local computer and on Appveyor CI.

The only change was replacing "eap-25" to "eap-52", and reverting back fixes build.

**Comment by ilmat192** [comment]
The root cause may be in [this](https://github.com/JetBrains/kotlin/commit/2a889ef3500aec91f33041406a9d35293e16938c) commit. It's strange because it worked fine on my machine and during CI runs.

**Comment by ilmat192** [comment]
Oh, I see, stupid mistake. [This](https://github.com/JetBrains/kotlin/commit/347bc2c04099372c8ba4b71791194f18ba7257e8) should fix it.

**Comment by msink** [comment]
Maybe, but I don't know how to test it.
Will be there next RC soon?

**Comment by olonho** [comment]
Only after NY, I afraid.

**Comment by msink** [comment]
Not critical for me, but wonder why there isn't a test for combination "multiplatform + cinterop + windows"

**Comment by msink** [comment]
Still not applied in both `master` and `1.3.20` branches?

**Comment by msink** [comment]
Tried 1.3.20-eap-77 - same error.

**Comment by msink** [comment]
Weird - this does not work too - gcc works, cinterop doesnt:
```
set "PATH=F:\work\.konan\dependencies\msys2-mingw-w64-x86_64-gcc-7.3.0-clang-llvm-lld-6.0.1\bin;%PATH%"
gcc --version
gradlew build
```

**Comment by ilmat192** [comment]
Sorry, wrong patch version. This works fine: https://github.com/JetBrains/kotlin/commit/4aaf36131f2461c024debcee85897b241a1db94d.

**Comment by msink** [comment]
Fixed in `1.3.30-dev-701`, so in next `1.3.20-eap` should be fixed too.

**Comment by ilmat192** [comment]
Fixed in `1.3.20-eap-87` too (available at the dev-repo).

**Comment by msink** [comment]
Tried 1.3.20-eap-87 - fixed.
Thanks.
